ethereum-consortium/docs/setupWalkthrough.md

```diff
@@ -87,11 +87,11 @@
       resource and copy the IP.  
 
       Navigate to http://{copied ip}:3001/enodes. Save the returned value
-      {bootnodes}.
```

Comment:
I was using boot nodes, but static nodes appears to increase reliability. They don't always have to be available, but the node will continuously attempt to connect to anything that is in the list. I'm sure there is a trade off. We could look through the code to see what it is at some point in time.